src/main/java/com/EighthLight/app/Config.java

```diff
@@ -45,11 +50,18 @@ private void setPlayers() {
             userInput = ui.getInput();
         }
         players.addAll(gameMode.get(userInput));
+        if (userInput.equals("2")) {
```

Comment:
Hmmm I wonder if you could make this more readable by setting "2" to a constant that stood for what game mode it is.  I think it might be hard to remember if this gets big enough what all the 1 and 2s mean